transformer/dockerfilegenerator/rubydockerfiletransformer.go

```diff
@@ -59,14 +59,14 @@ func (t *RubyDockerfileGenerator) GetConfig() (transformertypes.Transformer, *en
 
 // DirectoryDetect runs detect in each sub directory
 func (t *RubyDockerfileGenerator) DirectoryDetect(dir string) (map[string][]transformertypes.Artifact, error) {
-	gemfilePaths, err := common.GetFilesByName(dir, []string{"Gemfile"}, nil)
+	gemfilePaths, err := common.GetFilesInCurrentDirectory(dir, []string{"Gemfile"}, nil)
 	if err != nil {
 		return nil, fmt.Errorf("failed to look for Gemfiles in the dir %s . Error: %q", dir, err)
 	}
 	if len(gemfilePaths) == 0 {
 		return nil, nil
 	}
-	rubyFiles, err := common.GetFilesByExt(dir, []string{rubyFileExt})
+	rubyFiles, err := common.GetFilesByExtInCurrDir(dir, []string{rubyFileExt})
```

Comment:
Revert this to `GetFilesByExt`
We only reach this code if there is a Gemfile detected in the current folder.

Reply:
done.